### DISCUSSION TITLE
Taking NODE_ENV by default without checking it is risky and leads to errors and infinite frustrations

### DISCUSSION THREAD

**Discussion by ericjeker** [strapi/strapi#6963] [Legacy Topics]
**Describe the bug**

I followed the Quick-Start and finally arrived on the Strapi Dashboard but couldn't find the "+ Create new collection type" link anywhere. After a long discussion on Slack it seems that Strapi uses the local NODE_ENV variable which can be set to anything and there is no control whatsoever on it.

**Steps to reproduce the behavior**

1. Set your local NODE_ENV to "dev" or "toto"
2. npx create-strapi-app my-project --quickstart
3. Follow the steps to access the admin dashboard
4. See no link on the admin dashboard

**Expected behavior**

1. I'd like to know that you use that variable when following the Quick-Start
2. The content of the variable should be checked to make sure it match a list of allowed values and a useful error message returned when it doesn't
3. It would be even better to not use it and to set it by default when running `strapi develop`

**System**

- Node.js version: v12.14.1
- NPM version: 6.14.3
- Strapi version: 3.0.5
- Database: None
- Operating system: Windows

**Screenshots**

![image](https://user-images.githubusercontent.com/979397/87131846-9a3a1680-c2c7-11ea-8a6e-3759080494a2.png)
![image](https://user-images.githubusercontent.com/979397/87131868-a2925180-c2c7-11ea-9c4a-49a0971d3324.png)

**Comment by derrickmehaffy** [comment]
Hello,

I see you are wanting to ask a question that is not really a bug report or a feature request, questions should be directed to the [discussions](https://github.com/strapi/strapi/discussions). Please see the following contributing guidelines for asking a question [here](https://github.com/strapi/strapi/blob/master/CONTRIBUTING.md#reporting-an-issue).

I have converted this to a discussion for you, we try to keep Github clean and only for bug reports and feature requests.
